draft-ietf-jsonpath-base.md

```diff
@@ -482,14 +484,14 @@ stands for a nodelist that contains the root node of the argument,
 followed by a possibly empty sequence of *selectors*.
 
 ~~~~ abnf
-json-path = root-selector *(dot-selector        /
-                            dot-wild-selector   /
-                            index-selector      /
-                            index-wild-selector /
-                            union-selector      /
-                            slice-selector      /
-                            descendant-selector /
-                            filter-selector)
+json-path = root-selector *(S (dot-selector        /
+                               dot-wild-selector   /
+                               index-selector      /
+                               index-wild-selector /
+                               union-selector      /
+                               slice-selector      /
+                               descendant-selector /
+                               filter-selector))
```

Comment:
Do we really want to allow jsonpathes  scattered across several lines

```
$   .   a
[ 'b' ] 
[ 123 ] 

Has there been users demand for this ?

Reply:
You mean, allowing newlines in JSONPath?  Depends a bit on how long they are.  I seem to remember we wanted to be lenient here...

Reply:
yes i can remember darkly too ... so your edits all in all make sense.